resource_jenkins_job.go

```diff
@@ -44,12 +49,70 @@ func resourceJenkinsJob() *schema.Resource {
 					"URL pointing to a text file (http://... or https://...), or as filesystem URL (file://...).",
 				Required: true,
 			},
-			"parameters": {
+			"configuration": &schema.Schema{
```

Comment:
`parameters` can be confused by the Parameters section of the job, so it was renamed to `configuration`.